### PR TITLE
No longer applying volume when reading audio injector buffer

### PR DESCRIPTION
**Pull request by davidkelly**
Since we also apply the gain in HRTF (on both server and client), the mono sources sort of had volume^2 applied to them.  Pulled out the setVolume on the `AudioInjectorLocalBuffer` as well, since it is now unused.  Capped the volume (0-1) in `AudioInjector` now as well - was capped in the local buffer.

Test Plan:

Smoke test audio - a mono, stereo, and ambisonic injector should all probably be tested.  Perhaps Audio.playSound executed locally in the console first.  Then, just to be sure, have an entity or avatar execute them and listen on the other side, too.   
Some test sounds:
mono sin wave: https://s3-us-west-1.amazonaws.com/hifi-content/davidkelly/production/audio/sin220m.wav

stereo sin wave: https://s3-us-west-1.amazonaws.com/hifi-content/davidkelly/production/audio/sin220s.wav

ambisonic: http://hifi-content.s3.amazonaws.com/ken/test/horizontal_test_ambiX.wav

for mono and stereo, be sure to put in your current position so it plays where you are:
`Audio.playSound(ambisonicSound, {position: MyAvatar.position});`
for ambisonic, be sure to put in your orientation so right/left/front/back work as you would expect:
`Audio.playSound(ambisonicSound, {orientation: MyAvatar.orientation});`

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3f3d922ecaefaa94c2f791b28e3be419a20a91a2 available at: https://gist.github.com/2501e69920bf8dc2f01f63d552f3964b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9197-3f3d922.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9197-3f3d922.dmg

File: libraries/audio-client/src/AudioClient.cpp [hunk 1]
```diff
@@ -1132,7 +1132,7 @@ void AudioClient::mixLocalAudioInjectors(float* mixBuffer) {
 
                     // stereo gets directly mixed into mixBuffer
                     for (int i = 0; i < AudioConstants::NETWORK_FRAME_SAMPLES_STEREO; i++) {
-                        mixBuffer[i] += (float)_scratchBuffer[i] * (1/32768.0f);
+                        mixBuffer[i] += (float)_scratchBuffer[i] * (1/32768.0f) * injector->getVolume();
```
Comment (kencooke) [inline libraries/audio-client/src/AudioClient.cpp]: Better to hoist getVolume() out of the loop, using something like:
```
// no distance attenuation
float gain = (1/32768.0f) * injector->getVolume();
```

**Comment by hifi-gustavo** [comment]
Build console output for commit: d6b99ce57e53664f9108464200b29ca14c58964b available at: https://gist.github.com/2501e69920bf8dc2f01f63d552f3964b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9197-d6b99ce.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9197-d6b99ce.dmg

File: libraries/audio-client/src/AudioClient.cpp (hunk 1 above)
Comment (davidkelly) [inline libraries/audio-client/src/AudioClient.cpp]: agreed - done.

**Comment by hifi-gustavo** [comment]
Build console output for commit: a8157ae2cdaaa5e5dda1e69a75741032379543ae available at: https://gist.github.com/2501e69920bf8dc2f01f63d552f3964b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9197-a8157ae.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9197-a8157ae.dmg

**Comment by Nex-Pro** [comment]
### PASS

Ambisonic working well. 
Mono is very mono.
Stereo is fine.

Tested in localhost.

Also checked sounds at dev-demo and dev-welcome. Firework directionality at dev-playa.
reverb test and echo server audio.

Note: I was able to hear that the ambisonic sound was faintly audible at volumes in the range  0.00025  - 0.00015 with a less than silent workstation running next to me.

I tested sounds by using the in-world audio injector from the marketplace and via a script.

**Pull request merged**